examples/src/main/java/io/opencensus/examples/grpc/helloworld/HelloWorldServer.java

```diff
@@ -138,7 +143,7 @@ public static void main(String[] args) throws IOException, InterruptedException
       StackdriverStatsExporter.createAndRegister(
           StackdriverStatsConfiguration.builder()
               .setProjectId(cloudProjectId)
-              .setExportInterval(Duration.create(60, 0))
+              .setExportInterval(Duration.create(5, 0))
```

Comment:
Is this effectively the buffering delay before spans get exported? Why did you need to change it?

Reply:
This is for the stats exporter, it changes the aggregation interval, so you get higher resolution. Thought it might be nice for the example, but happy to change it back too.